product/models.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length = 50)
+
+    class Meta:
+        db_table = 'categories'
+
+class Collection(models.Model):
+    name  = models.CharField(max_length = 50)
+    image = models.URLField(max_length = 500)
+    video = models.URLField(max_length = 500)
+
+    class Meta:
+        db_table = 'collections'
+
+class Product(models.Model):
+    category          = models.ForeignKey(Category, on_delete = models.CASCADE)
```

Comment:
ForeignKey 첫벗째 인자를 통일되게 string으로 변경해주세요!
```suggestion
    category          = models.ForeignKey('Category', on_delete = models.CASCADE)
```